modules/local_contacts/modules.php

```diff
@@ -23,7 +23,7 @@ public function process() {
             if (!empty($addresses)) {
                 $contacts = $this->get('contact_store');
                 foreach ($addresses as $vals) {
-                    $contacts->add_contact(array('source' => 'local', 'email_address' => $vals['email'], 'display_name' => $vals['name']));
+                    $contacts->add_contact(array('source' => 'local', 'email_address' => $vals['email'], 'display_name' => $vals['name'], 'group' => isset($vals['contact_group']) ? $vals['contact_group'] : 'Personal Addresses'));
```

Comment:
We have 3 more contact stores - carddav, ldap, gmail. Shouldn't we extend those as well to support new functionality?

Reply:
Working on it.

Reply:
@kroky While waiting to have the identifiers to test other contact stores: carddav, ldap, gmail, can you merge this one and we will then do another PR for the rest ?

Reply:
I am fine with separate PRs for the rest.